src/style/index.css

```diff
@@ -0,0 +1,90 @@
+*{
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  /*overflow-x: hidden;*/
+}
+
+body{
+  font-family: Montserrat, sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  display: flex;
+}
+
+header{
+  margin: 0;
+  padding: 0;
+}
+
+header h1 b{
+  color: #2CC4D2;
+}
+
+main{
+  display: flex;
+  padding: 100px 10%;
```

Comment:
Please consider changing it. On large screens it looks not so good. Typically 
![image](https://user-images.githubusercontent.com/14924742/81448784-a6024300-9187-11ea-8a9c-4ac923fae909.png)